src/improvements/tasks/sep/019-worldchain-l1-ownership-transfers/README.md

```diff
@@ -1,6 +1,6 @@
 # 019-worldchain-l1-ownership-transfers: Transfer L1 owners for Worldchain Sepolia (DGF, PermissionlessWETH, PermissionedWETH and L1PAO)
 
-Status: [READY TO SIGN]()
+Status: [CANCELLED]()
```

Comment:
Cancelling this because another task will supersede it.

Reply:
make sense!